README.md

```diff
@@ -90,6 +90,21 @@ See the
 [scan result object](https://github.com/sodadata/soda-sql/blob/main/core/sodasql/scan/scan_result.py)
 for all attributes and methods.
 
+Or, return Spark data frames:
+
+``` python
```

Comment:
@anilkulkarni87 : I have added an example to the docs for returning the data frames. This will also be shown on PyPI.